### ISSUE TITLE
[`playwright/no-conditional-in-test`]: Avoid having conditionals in tests

### ISSUE THREAD

**Issue by splincode** [mskelton/eslint-plugin-playwright#406]
```ts
const {x = 0, y = 0, width = 0, height = 0} = (await button.boundingBox()) ?? {};
```

<img width="1264" height="587" alt="Image" src="https://github.com/user-attachments/assets/ccd9f649-f411-4a94-b254-759d717196c9" />

Is it false positive?

**Comment by splincode** [comment]
`const {x = 0, y = 0, width = 0, height = 0} = (await button.boundingBox()) || {};` 

Could you check that case please
